java/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,18 @@
+# Set to debug or trace if log4j initialization is failing
+status = warn
+
+# Console appender configuration
+appender.console.type = Console
+appender.console.name = consoleLogger
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} [%tn] %-5p %c{1}:%L - %m%n
```

Comment:
Perhaps let's update this with the pattern containing more info? https://github.com/restatedev/sdk-java#logging

Reply:
hmm, I saw this a bit late... If I change this, I need to change all the log output in the tour... Can I leave it for now?

Reply:
we can leave it for the time being.